election/Makefile

```diff
@@ -1,17 +1,22 @@
 all: push
 
 # 0.0 shouldn't clobber any released builds
-TAG = 0.0
+TAG = 0.5
```

Comment:
Comment above is irrelevent if we want to make this change. How about
```Makefile
# 0.0 shouldn't clobber any released builds
# current latest is 0.5
TAG ?= 0.0
```
So tag will needed to be provided explicitly to deploy new version to gcr.io.